### PR TITLE
Refactor test logic for Audio and Midi into their own classes

### PR DESCRIPTION
**Pull request by BenjaminFair**

### CODE REVIEW

File: android/WALT/app/src/main/java/org/chromium/latency/walt/MidiTest.java [hunk 1]
```diff
@@ -0,0 +1,228 @@
+package org.chromium.latency.walt;
```
Comment (kamrik) [inline android/WALT/app/src/main/java/org/chromium/latency/walt/MidiTest.java]: Copyright header

Comment (BenjaminFair) [inline android/WALT/app/src/main/java/org/chromium/latency/walt/MidiTest.java]: Done

**Comment by kamrik** [comment]
Looks good, merging.

**Comment by gkasten** [comment]
+1

**Pull request merged**